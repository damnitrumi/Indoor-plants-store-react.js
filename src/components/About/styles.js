import styled, { css } from "styled-components";
import { Container as SectionWrapper } from "../SectionWrapper/styles";
import { Container as Text } from "../Text/styles";

export const Container = styled.div`
  ${() => css`
    counter-reset: TextCounter;
    ${SectionWrapper} {
      display: flex;
      flex-direction: column;
      gap: 180px;
    }
  `}
`;
export const AboutContainer = styled.div`
  ${({ theme }) => css`
    ${Text}::before {
      content: counter(TextCounter) ".";
      counter-increment: TextCounter;
      font-size: ${theme.font.sizes.large};
      font-family: ${theme.font.family.primary};
      margin-right: ${theme.spacings.small};
      color: ${theme.colors.lightGreen};
    }

    padding: ${theme.spacings.small} 0;
    /* margin-top: 180px; */
    overflow-y: auto;
  `}
`;
