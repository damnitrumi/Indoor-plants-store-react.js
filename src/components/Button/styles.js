import styled, { css } from "styled-components";

export const Container = styled.button`
  ${({ theme }) => css`
    background: transparent;
    border: none;
    cursor: pointer;
    padding: ${theme.spacings.tiny};
    color: white;
    position: relative;
  `}
`;
