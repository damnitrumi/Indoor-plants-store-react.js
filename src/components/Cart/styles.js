import styled, { css } from "styled-components";

export const Container = styled.div`
  ${() => css`
    width: 360px;
  `}
`;

export const ProductContainer = styled.div`
  ${() => css``}
`;

export const ProductImageContainer = styled.div`
  ${() => css``}
`;
export const ProductData = styled.div`
  ${() => css``}
`;

export const NameContainer = styled.div`
  ${() => css`
    display: flex;
    justify-content: space-between;
  `}
`;

export const ProductOptionsContainer = styled.div`
  ${() => css``}
`;

export const QuantityContainer = styled.div`
  ${() => css``}
`;
