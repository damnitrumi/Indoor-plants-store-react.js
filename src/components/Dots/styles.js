import styled, { css } from "styled-components";

export const Container = styled.span`
  ${({ theme, current, index }) => css`
    display: inline-block;
    border-radius: 50%;
    height: ${theme.spacings.tiny};
    width: ${theme.spacings.tiny};
    background-color: ${current == index
      ? theme.colors.grey
      : theme.colors.mediumGrey};
  `}
`;
