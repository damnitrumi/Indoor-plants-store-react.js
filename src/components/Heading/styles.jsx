import styled, { css } from "styled-components";

export const Container = styled.h1`
  ${({ theme }) => css`
    font-family: ${theme.font.family.primary};
    color: ${theme.colors.darkGreen};
    margin-bottom: ${theme.spacings.medium};
    font-size: ${theme.font.sizes.large};
    text-align: center;
  `}
`;
