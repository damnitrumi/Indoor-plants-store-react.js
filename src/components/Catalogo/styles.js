import styled, { css } from "styled-components";
import { Container as SectionWrapper } from "../SectionWrapper/styles";
import { Container as ImageContainer } from "../ImageContainer/styles";

// export const SectionContainer = styled.div`
//   ${() => css`
//     display: flex;
//     flex-direction: column;
//     height: 100%;
//   `}
// `;

export const Container = styled.div`
  ${({ visible }) => css`
    height: 100vh;
    display: flex;
    flex-direction: column;
    transition: transform 0.5s;
    transform: translateX(-100%);
    position: absolute;
    width: 360px;
    ${visible &&
    css`
      transform: translateX(0);
    `}
    ${SectionWrapper} {
      flex: 1;
      display: flex;
      flex-direction: column;
    }
  `}
`;

export const SlidesContainer = styled.div`
  ${() => css`
    /* ${ImageContainer} {
      height: 460px;
    } */
  `}
`;

export const SliderContainer = styled.div`
  ${() => css`
    flex: 1;
  `}
`;

export const BgContainer = styled.span`
  ${({ theme }) => css`
    display: block;
    background-color: ${theme.colors.lightGrey};
    width: 230px;
    height: 300px;
    border-radius: ${theme.spacings.medium};
    position: absolute;
    z-index: -1;
    top: 40px;
  `}
`;

export const DotsContainer = styled.div`
  ${() => css`
    display: flex;
    justify-content: center;
    gap: 3px;
    position: absolute;
    left: 150px;
    bottom: 290px;
  `}
`;
