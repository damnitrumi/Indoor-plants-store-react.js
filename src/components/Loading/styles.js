import styled, { css } from "styled-components";
import { Container as SectionWrapper } from "../SectionWrapper/styles";

export const Container = styled.div`
  ${() => css`
    ${SectionWrapper} {
      display: flex;
      align-items: center;
      justify-content: center;
    }
  `}
`;
