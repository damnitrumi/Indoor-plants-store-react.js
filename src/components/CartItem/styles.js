import styled, { css } from "styled-components";

import { Container as Heading } from "../Heading/styles";
import { Container as Button } from "../Button/styles";

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    margin-top: ${theme.spacings.small};
    ${Heading} {
      margin-bottom: 0;
      font-size: 1.4rem;
      font-family: ${theme.font.family.secondary};
    }
    ${Button} {
      color: initial;
    }
  `}
`;

export const ProductImageContainer = styled.div`
  ${() => css`
    flex: 1;
  `}
`;
export const ProductData = styled.div`
  ${({ theme }) => css`
    flex: 2;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    padding-right: ${theme.spacings.large};
  `}
`;

export const NameContainer = styled.div`
  ${() => css`
    display: flex;
    justify-content: space-between;
  `}
`;

export const ProductOptionsContainer = styled.div`
  ${() => css``}
`;

export const QuantityContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    justify-content: space-between;
    font-size: ${theme.font.sizes.medium};
  `}
`;
export const Quantity = styled.div`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    border: 1px solid rgba(0, 0, 0, 0.3);
    ${Button}:first-of-type {
      border-right: 1px solid rgba(0, 0, 0, 0.3);
    }

    ${Button}:last-of-type {
      border-left: 1px solid rgba(0, 0, 0, 0.3);
    }

    span {
      padding: 0 ${theme.spacings.tiny};
    }
  `}
`;
