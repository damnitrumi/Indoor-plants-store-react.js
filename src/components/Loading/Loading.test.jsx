import { screen } from "@testing-library/react";
import { Loading } from ".";
import { renderTheme } from "../../styles/render-theme";

describe("Loading", () => {
  it("should render Loading component", () => {
    renderTheme(<Loading />);
    expect(screen.getByRole("img")).toBeInTheDocument();
  });

  it("should match snapshot", () => {
    const { container } = renderTheme(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
