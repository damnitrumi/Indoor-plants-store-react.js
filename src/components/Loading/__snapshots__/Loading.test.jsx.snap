// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match snapshot 1`] = `
.c2 {
  height: 100vh;
  border-radius: 5px;
  background: white;
  background: url(/assets/images/background-home.png);
  background-repeat: no-repeat;
  background-size: cover;
  padding: 3.2rem;
}

.c0 .c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1 c2"
    >
      <img
        class=""
        src="/assets/images/logo-light.png"
      />
    </div>
  </div>
</div>
`;
